### DISCUSSION TITLE
Can we support two rounds of LED around TV/Monitor for eliminating the gap between two consecutive LEDs in case of 30 LED/m or 60 LED/m

### DISCUSSION THREAD

**Discussion by mkunjir11** [awawa-dev/HyperHDR#668] [Ideas]
Hello,
I am pretty new to Hyper HDR.
The issue I am having is with the resolution of LED.
I have a 27 inch monitor (61cm x 36m)
My LED strip is WS2812b 60 LEDs/m - 1 LED for every 1.2cm - 5 mtr spool

Length wise my monitor can hold only 39 LEDs, breadthwise its 23 LEDs.
However a LED frame of 39 X 23 LEDS yeilds a very low resolution. ( I am aware of 144 LED/m strips but those are expensive )

Morever, due to this, I am using only 124 LEDs (TOP 39 + BOTTOM 39 + LEFT 23 + RIGHT 23 = 124), out of total 300.

Suggestion:
can we have two rounds of LED behind the Monitor/TV instead of just 1? (Or is there a solution in the software already to address this, I am not aware)

This way the second round can take place of the gaps the first round has.

Essentially in this scenario the first LED will always be the first LED from first round, but the second LED will be the first LED from the second round (in my case the 125th LED) and so on ...

By two rounds I don't mean two physically seperate LED strips, but one single LED strip that goes around the TV/Monitor twice

**Comment by awawa-dev** [comment]
Hi
I think it is possible if you manage to cover the monitor with this LED strip twice. Otherwise, it is recommended to connect Pico and multi-segment to one RP2040: one inner segment and one outer segment. The challenge for both solutions will be manual editing of the LED strip geometry in HyperHDR. There is no wizard that can do such things because this is an unusual case, but you can e.g. create the first segment using the wizard, copy it, create the second segment also using the wizard and edit &combine both configurations into one. Nevertheless, it is a challenge.

**Comment by mkunjir11** [comment]
@awawa-dev I see what you mean, I will try this. I understand this is an unusual edge case and can't be a feature request, just wanted to be sure that I am not missing anything from the software that could address this.

Also, if the approach of creating two strips succeeds, can't I continue to use ESP8266 with HyperSerial to drive them, given that ESP8266 can drive all of the 300 pixels in a straight line?

**Reply by awawa-dev** [reply]
No, esp8266 hardware limitation does not allow multiple segments to be used with i2s parallel rendering. Theoretically you can try WLED and its segments, but without i2s parallel output synchronization will not be guaranteed.
